requirements.txt

```diff
@@ -1,7 +1,7 @@
-setuptools==50.3.0
-zc.buildout==2.13.3
-wheel
+setuptools==52.0.0
+zc.buildout==3.0.a2
```

Comment:
I guess this should be 3.0.0a2. Do we really want to depend on a zc.buildout alpha? That should be fine for the 6.0 branch though...

Reply:
Looks like it works despite the typo.

This is primary a quick test to see if it works. And it targets the 6.0 branch anyway.